src/ast/node/function/generator.rs

```diff
@@ -202,7 +202,7 @@ pub(crate) fn init_generator<'a>(
         modifier: Some((TokenType::PUB, Default::default())),
         body_range: Default::default(),
         is_trait: false,
-        is_tuple: true,
+        is_tuple: false,
```

Comment:
[Trivial]: What for the changes here? Could you add a comment for `STType`?

Reply:
It's probably a typo, a generator is not a tuple